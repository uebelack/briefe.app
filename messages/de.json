{
  "common": {
    "title": "Briefe App",
    "title_long": "Briefe App - Mühelos professionelle Briefe auf iPhone, iPad, Mac und Android-Geräten erstellen",
    "scroll_down": "Nach unten scrollen",
    "language": "Deutsch",
    "download": "Herunterladen",
    "android_only": "* Aktuell nicht verfügbar auf Android-Geräten",
    "ios_only": "** Aktuell nur auf iPhone und iPad verfügbar"
  },
  "home": {
    "teaser": "Im Handumdrehen professionell formatierte Briefe erstellen und drucken."
  },
  "details": {
    "title": "Details",
    "p1": "Selbst in einer Zeit digitaler Kommunikation und sofortiger Vernetzung bleiben traditionelle Briefe eine unverzichtbare Form des Austauschs. Warum? Weil sie eine zeitlose und authentische Art der Korrespondenz darstellen. Ob für persönliche oder geschäftliche Nachrichten – Briefe vermitteln eine Tiefe und Bedeutung, die in E-Mails oder Textnachrichten oft verloren geht. Sie sind ein Ausdruck von Sorgfalt und Aufmerksamkeit und hinterlassen einen bleibenden Eindruck. Unsere Brief-App kombiniert diese zeitlose Tradition mit modernem Komfort und ermöglicht es Ihnen, professionelle Briefe einfach zu erstellen, zu teilen und zu verwalten.",
    "p2": "Die Briefe App ermöglicht es Ihnen, professionell formatierte Briefe schnell und mühelos auf iPhone, iPad, Mac und Android-Geräten zu erstellen und zu drucken. Ihre Briefe erhalten ein ansprechendes Layout, das einen professionellen Eindruck hinterlässt. Mit nur einem Klick können Sie Ihre Briefe direkt aus der App drucken. Sie haben auch die Möglichkeit, Ihre Briefe digital zu teilen oder als PDF zu exportieren, um sie auf verschiedenen Plattformen zu verteilen. Die App synchronisiert sich nahtlos über iCloud auf all Ihren Geräten, sodass Sie jederzeit und überall auf Ihre Briefe zugreifen können. Dank der einfachen Suchfunktion lassen sich bereits verfasste Briefe schnell wiederfinden.",
    "pro_title": "Mit Pro die KI-gestützte Briefassistenz freischalten",
    "p3": "Steigen Sie auf die Pro-Version um und heben Sie Ihr Briefschreiben auf ein neues Level mit unserem KI-Assistenten! Optimieren Sie mühelos Struktur, Klarheit und Ton Ihrer Briefe, während die ursprüngliche Intention erhalten bleibt. Egal, ob Sie einen Entwurf verbessern, einen vollständigen Brief nur anhand eines Betreffs generieren oder einfach Inspiration suchen – die KI sorgt für professionelle und perfekt formulierte Ergebnisse. Sie unterstützt mehrere Sprachen und passt sich Ihrem Schreibstil an, um Ihre Korrespondenz noch einfacher zu gestalten.",
    "p4": "Die Pro-Version bietet zudem weitere Premium-Funktionen wie das Hinzufügen persönlicher Unterschriften direkt in der App, das mühelose Importieren von Kontaktadressen aus Ihrem Adressbuch und eine große Auswahl an Vorlagen für verschiedene Anlässe. Darüber hinaus können Sie das Format Ihrer Briefe individuell anpassen.",
    "p5": "Die Briefe App vereinfacht Ihre Korrespondenz und ermöglicht es Ihnen, Briefe mühelos zu erstellen, zu teilen und zu drucken – und das mit Stil, Klarheit und Benutzerfreundlichkeit."
  },
  "features": {
    "title": "Funktionen",
    "description": "Um Ihnen einen umfassenden Überblick zu verschaffen, haben wir alle Funktionen im Detail aufgelistet, und zwar entsprechend ihrer Verfügbarkeit in den verschiedenen App-Versionen. So können Sie leicht erkennen, welche Features jede Version der App bietet.",
    "format": {
      "title": "Automatische Formatierung",
      "description": "Briefe werden automatisch professionell formatiert"
    },
    "print": {
      "title": "Drucken",
      "description": "Drucken Sie Ihre Briefe bequem direkt aus der App"
    },
    "export": {
      "title": "Teilen & Exportieren",
      "description": "Teilen Sie Ihre Briefe digital oder exportieren Sie sie als PDF-Dateien"
    },
    "sync": {
      "title": "Speicherung",
      "description": "Ihre Briefe werden über die iCloud geräteübergreifend synchronisiert"
    },
    "signatures": {
      "title": "Unterschriften",
      "description": "Fügen Sie persönliche Unterschriften direkt in der App hinzu"
    },
    "contacts": {
      "title": "Kontakte",
      "description": "Verwenden Sie Ihre Kontaktadressen direkt aus Ihrem Adressbuch"
    },
    "templates": {
      "title": "Vorlagen",
      "description": "Nutzen Sie eine breite Palette von Vorlagen für verschiedene Gelegenheiten"
    },
    "layout": {
      "title": "Layout",
      "description": "Passen Sie die Formatierung Ihrer Briefe individuell an Ihre Bedürfnisse an"
    }
  },
  "faq": {
    "menu": "FAQ",
    "title": "Häufig gestellte Fragen"
  },
  "contact": {
    "title": "Kontakt",
    "subtitle": "Bitte kontaktieren Sie mich, wenn Sie Fragen haben",
    "email_placeholder": "Ihre E-Mail-Adresse",
    "subject_placeholder": "Ihr Anliegen",
    "message_placeholder": "Ihre Nachricht",
    "privacy_1": "Ich akzeptiere die",
    "privacy_2": "Datenschutzerklärung",
    "privacy_link": "/de/datenschutz",
    "privacy_info": "Ihre E-Mail-Adresse und Ihre Nachricht werden gespeichert, damit ich darauf antworten kann. Ihre Daten werden nicht für andere Zwecke verwendet.",
    "submit": "Senden",
    "email_error": "Bitte geben Sie eine gültige E-Mail-Adresse ein",
    "subject_error": "Bitte geben Sie Ihr Anliegen ein",
    "message_error": "Bitte geben Sie eine Nachricht ein",
    "captcha_error": "Bitte lösen Sie das Captcha",
    "privacy_error": "Bitte akzeptieren Sie die Datenschutzerklärung",
    "success": "Vielen Dank für Ihre Nachricht! Wir werden uns so schnell wie möglich bei Ihnen melden!",
    "error": "Leider ist ein unerwarteter Fehler aufgetreten. Bitte versuchen Sie es später erneut oder senden Sie uns eine E-Mail an support@briefe.app.",
    "cookie": "Bitte akzeptieren Sie die Cookies am unteren Rand des Bildschirms, um eine Nachricht zu übermitteln oder senden Sie eine E-Mail an support@briefe.app."
  },
  "footer": {
    "rights": "Alle Rechte vorbehalten.",
    "imprint": "Impressum",
    "imprint_link": "/de/impressum",
    "privacy": "Datenschutzerklärung",
    "privacy_link": "/de/datenschutz"
  },
  "cookie_consent": {
    "message": "Diese Webseite verwendet Cookies, um Ihnen einen besseren Service bieten zu können. Um mehr zu erfahren, lesen Sie bitte unsere",
    "accept": "Akzeptieren"
  },
  "lost": {
    "title": "Verloren!? 404 - Nicht gefunden",
    "description": "Die Seite, die Sie suchen, existiert nicht.",
    "button": "Startseite"
  },
  "help": {
    "menu": "Hilfe",
    "link": "hilfe",
    "title": "Hilfe",
    "how_to_title": "So schreiben Sie einen Brief mit der Briefe-App",
    "how_to_001": "Um mit dem Verfassen Ihres Briefes zu beginnen, tippen Sie einfach auf die Plus-Schaltfläche in der oberen rechten Ecke des Hauptbildschirms. Sobald Sie auf die Schaltfläche getippt haben, öffnet sich ein neuer Bildschirm, in dem Sie Ihren Brief ganz einfach verfassen können.",
    "how_to_002": "Geben Sie Absender- und Empfängeradresse ein, fügen Sie einen Betreff hinzu und verfassen Sie Ihre Nachricht im Feld 'Nachricht'. Sobald Sie Ihren Brief fertiggestellt haben, tippen Sie auf 'Speichern' in der oberen rechten Ecke.",
    "how_to_003": "Ihr Brief wird automatisch formatiert und angezeigt. Nun können Sie ihn entweder ausdrucken, indem Sie auf das Druckersymbol tippen, oder ihn über die Schaltfläche 'Teilen' (das quadratische Symbol mit einem Pfeil) per E-Mail versenden."
  },
  "blog": {
    "menu": "Blog",
    "title": "Blog - Gedanken, Geschichten und Ideen"
  },
  "call_to_action": {
    "text": "Laden Sie jetzt die kostenlose Briefe App herunter und schreiben Sie Ihren ersten Brief!"
  }
}