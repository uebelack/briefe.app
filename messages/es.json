{
  "common": {
    "title": "Aplicación de Cartas",
    "title_long": "Aplicación de Cartas - Crea cartas profesionales sin esfuerzo en iPhone, iPad, Mac y dispositivos Android",
    "scroll_down": "Desplázate hacia abajo",
    "language": "Español",
    "download": "Descargar",
    "android_only": "* actualmente no disponible en dispositivos Android",
    "ios_only": "** actualmente sólo en iPhone y iPad"
  },
  "home": {
    "teaser": "Crea e imprime cartas con formato perfecto en poco tiempo."
  },
  "details": {
    "title": "Detalles",
    "p1": "Incluso en una era dominada por la comunicación digital y la conectividad instantánea, las cartas tradicionales siguen siendo una forma esencial de correspondencia. ¿Por qué? Porque representan un modo de comunicación atemporal y auténtico. Ya sea para mensajes personales o profesionales, las cartas transmiten una profundidad y significado que a menudo se pierde en los correos electrónicos o mensajes de texto. Son una expresión de cuidado y atención, dejando una impresión duradera. La Aplicación de Cartas combina esta tradición con la comodidad moderna, permitiéndote crear, compartir y gestionar cartas profesionales de manera sencilla.",
    "p2": "La Aplicación de Cartas te permite redactar y imprimir cartas con un formato profesional de manera rápida y sencilla en iPhone, iPad, Mac y dispositivos Android. Tus cartas tendrán un diseño atractivo que dejará una impresión profesional. Con un solo clic, puedes imprimir directamente desde la app. También tienes la opción de compartir tus cartas digitalmente o exportarlas como archivos PDF para distribuirlas en diversas plataformas. La aplicación se sincroniza perfectamente a través de iCloud en todos tus dispositivos, permitiéndote acceder a tus cartas en cualquier momento y lugar. Gracias a su función de búsqueda intuitiva, puedes encontrar fácilmente cartas previamente redactadas.",
    "pro_title": "Desbloquea la Asistencia de Cartas con IA en la versión Pro",
    "p3": "¡Lleva tu redacción al siguiente nivel con nuestro Asistente de IA en la versión Pro! Mejora la estructura, claridad y tono de tus cartas de manera sencilla, manteniendo siempre la intención original. Ya sea que necesites perfeccionar un borrador, generar una carta completa a partir de un asunto o simplemente encontrar inspiración, la inteligencia artificial garantiza resultados profesionales y bien elaborados. Además, admite varios idiomas y se adapta a tu estilo de escritura para que la correspondencia sea más fácil y efectiva.",
    "p4": "La versión Pro también desbloquea funciones premium adicionales, como la posibilidad de añadir firmas personales directamente en la aplicación, importar direcciones de contacto desde tu libreta de direcciones y utilizar una amplia variedad de plantillas para distintas ocasiones. Además, puedes personalizar el formato de tus cartas según tus necesidades específicas.",
    "p5": "La Aplicación de Cartas simplifica tu correspondencia, permitiéndote crear, compartir e imprimir cartas de manera fácil, manteniendo siempre un estilo claro, profesional y fácil de usar."
  },
  "features": {
    "title": "Características",
    "description": "Para brindarte una visión completa, hemos enumerado todas las características en detalle, según su disponibilidad en las diferentes versiones de la aplicación. De esta manera, puedes ver fácilmente qué características ofrece cada versión de la aplicación.",
    "format": {
      "title": "Auto Formato",
      "description": "Las cartas se formatean automáticamente de manera profesional"
    },
    "print": {
      "title": "Imprimir",
      "description": "Imprime fácilmente tus cartas directamente desde la aplicación"
    },
    "export": {
      "title": "Compartir y Exportar",
      "description": "Comparte tus cartas de forma digital o exporta archivos PDF"
    },
    "sync": {
      "title": "Almacenamiento",
      "description": "Tus cartas se sincronizan en tus dispositivos usando iCloud"
    },
    "signatures": {
      "title": "Firmas",
      "description": "Agrega firmas personales directamente en la aplicación"
    },
    "contacts": {
      "title": "Contactos",
      "description": "Incorpora fácilmente direcciones de contacto desde tu libreta de direcciones"
    },
    "templates": {
      "title": "Plantillas",
      "description": "Utiliza una amplia gama de plantillas para diversas ocasiones"
    },
    "layout": {
      "title": "Diseño",
      "description": "Personaliza el diseño de tus cartas según tus necesidades específicas"
    }
  },
  "faq": {
    "menu": "FAQ",
    "title": "Preguntas más frecuentes"
  },
  "contact": {
    "title": "Contacto",
    "subtitle": "Por favor, contáctame si tienes alguna pregunta",
    "email_placeholder": "Tu dirección de correo electrónico",
    "subject_placeholder": "Tu asunto",
    "message_placeholder": "Tu mensaje",
    "privacy_1": "Acepto la",
    "privacy_2": "política de privacidad",
    "privacy_link": "/es/privacy",
    "privacy_info": "Tu dirección de correo electrónico y mensaje se almacenarán para que pueda responder. Tus datos no se utilizarán para otros fines.",
    "submit": "Enviar",
    "email_error": "Por favor, ingresa una dirección de correo electrónico válida",
    "subject_error": "Por favor, ingresa un asunto",
    "message_error": "Por favor, ingresa un mensaje",
    "captcha_error": "Por favor, resuelve este captcha",
    "privacy_error": "Por favor, acepta la política de privacidad",
    "success": "¡Gracias por tu mensaje! Nos pondremos en contacto contigo lo antes posible.",
    "error": "Lamentablemente, ocurrió un error inesperado. Por favor, inténtalo de nuevo más tarde o envíanos un correo electrónico a support@letter-app.com.",
    "cookie": "Por favor, acepta las cookies en la parte inferior de la pantalla para enviar un mensaje o enviar un correo a support@letter-app.com."
  },
  "footer": {
    "rights": "todos los derechos reservados.",
    "imprint": "Aviso Legal",
    "imprint_link": "/es/imprint",
    "privacy": "Política de Privacidad",
    "privacy_link": "/es/privacy"
  },
  "cookie_consent": {
    "message": "Este sitio web utiliza cookies para garantizar que obtengas la mejor experiencia en nuestro sitio web. Para obtener más información, consulta nuestra",
    "accept": "Aceptar"
  },
  "lost": {
    "title": "¡Estás perdido! 404 - No encontrado",
    "description": "La página que buscas no existe",
    "button": "Ir a Inicio"
  },
  "help": {
    "menu": "Ayuda",
    "link": "ayuda",
    "title": "Ayuda",
    "how_to_title": "Cómo escribir una carta con la aplicación de cartas",
    "how_to_001": "Para empezar a redactar tu carta, simplemente pulsa el botón más en la esquina superior derecha de la pantalla principal. Una vez que hayas pulsado el botón, se abrirá una nueva pantalla en la que podrás redactar fácilmente tu carta",
    "how_to_002": "Introduce la dirección del remitente y del destinatario, añade un asunto y redacta tu mensaje en el campo \"Mensaje\". Una vez que haya terminado su carta, pulse \"Guardar\" en la esquina superior derecha",
    "how_to_003": "Su carta será formateada y mostrada automáticamente. Ahora puede imprimirla pulsando el icono de la impresora o enviarla por correo electrónico utilizando el botón \"Compartir\" (el icono cuadrado con una flecha)"
  },
  "templates": {
    "menu": "Plantillas",
    "title": "Plantillas"
  },
  "blog": {
    "menu": "Blog",
    "title": "Blog - Pensamientos, historias e ideas"
  },
  "call_to_action": {
    "text": "Descarga la Aplicación de Cartas ahora y empieza a escribir tu primera carta — ¡es gratis!"
  }
}