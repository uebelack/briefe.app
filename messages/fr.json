{
  "common": {
    "title": "Application de Lettres",
    "title_long": "Application de Lettres - Créez sans effort des lettres professionnelles sur iPhone, iPad, Mac et appareils Android",
    "scroll_down": "Faites défiler vers le bas",
    "language": "Français",
    "download": "Télécharger",
    "android_only": "* actuellement non disponible sur les appareils Android",
    "ios_only": "** actuellement uniquement sur iPhone et iPad"
  },
  "home": {
    "teaser": "Créez et imprimez des lettres parfaitement formatées en un rien de temps."
  },
  "details": {
    "title": "Détails",
    "p1": "Même à l’ère de la communication numérique et de la connectivité instantanée, les lettres traditionnelles restent un moyen de correspondance incontournable. Pourquoi ? Parce qu’elles représentent une forme d’échange intemporelle et authentique. Qu’il s’agisse de messages personnels ou professionnels, une lettre transmet une profondeur et une signification souvent perdues dans les e-mails ou les messages instantanés. C’est une expression de soin et d’attention qui laisse une impression durable. L’Application de Lettres allie cette tradition à la modernité, vous permettant de rédiger, partager et gérer des lettres professionnelles en toute simplicité.",
    "p2": "L’Application de Lettres vous permet de créer et d’imprimer des lettres au format professionnel rapidement et facilement sur iPhone, iPad, Mac et appareils Android. Vos lettres bénéficient d’une mise en page soignée qui laisse une impression professionnelle. En un clic, vous pouvez les imprimer directement depuis l’application. Vous avez également la possibilité de les partager numériquement ou de les exporter en PDF pour les envoyer sur différentes plateformes. L’application se synchronise parfaitement via iCloud sur tous vos appareils, vous permettant d’accéder à vos lettres à tout moment et où que vous soyez. Grâce à sa fonction de recherche intuitive, retrouvez facilement vos lettres précédemment rédigées.",
    "pro_title": "Débloquez l’Assistant IA avec la version Pro",
    "p3": "Passez à la version Pro et profitez de l’Assistant IA pour améliorer vos lettres ! Affinez la structure, la clarté et le ton de vos courriers tout en conservant leur intention d’origine. Que vous souhaitiez perfectionner un brouillon, générer une lettre complète à partir d’un sujet ou simplement trouver l’inspiration, l’intelligence artificielle garantit un résultat professionnel et bien rédigé. Adapté à plusieurs langues et à votre style d’écriture, il facilite votre correspondance comme jamais auparavant.",
    "p4": "La version Pro débloque également des fonctionnalités premium supplémentaires, telles que l’ajout de signatures personnelles directement dans l’application, l’importation facile d’adresses depuis votre carnet de contacts et l’accès à une large gamme de modèles pour différentes occasions. Vous pouvez aussi personnaliser le format de vos lettres selon vos besoins spécifiques.",
    "p5": "L’Application de Lettres simplifie votre correspondance en vous permettant de rédiger, partager et imprimer vos lettres facilement, tout en garantissant un style clair, professionnel et convivial."
  },
  "features": {
    "title": "Fonctionnalités",
    "description": "Pour vous donner un aperçu complet, nous avons répertorié toutes les fonctionnalités en détail, en fonction de leur disponibilité dans les différentes versions de l'application. Vous pouvez ainsi voir facilement quelles fonctionnalités chaque version de l'application propose.",
    "format": {
      "title": "Mise en forme automatique",
      "description": "Les lettres sont automatiquement mises en forme de manière professionnelle"
    },
    "print": {
      "title": "Imprimer",
      "description": "Imprimez facilement vos lettres directement depuis l'application"
    },
    "export": {
      "title": "Partager et Exporter",
      "description": "Partagez vos lettres numériquement ou exportez-les au format PDF"
    },
    "sync": {
      "title": "Stockage",
      "description": "Vos lettres sont synchronisées sur vos appareils via iCloud"
    },
    "signatures": {
      "title": "Signatures",
      "description": "Ajoutez des signatures personnelles directement dans l'application"
    },
    "contacts": {
      "title": "Contacts",
      "description": "Intégrez facilement des adresses de contact depuis votre carnet d'adresses"
    },
    "templates": {
      "title": "Modèles",
      "description": "Utilisez une large gamme de modèles pour différentes occasions"
    },
    "layout": {
      "title": "Mise en page",
      "description": "Personnalisez la mise en page de vos lettres selon vos besoins spécifiques"
    }
  },
  "faq": {
    "menu": "FAQ",
    "title": "Questions fréquemment posées"
  },
  "contact": {
    "title": "Contact",
    "subtitle": "N'hésitez pas à me contacter si vous avez des questions",
    "email_placeholder": "Votre adresse e-mail",
    "subject_placeholder": "Votre sujet",
    "message_placeholder": "Votre message",
    "privacy_1": "J'accepte la",
    "privacy_2": "politique de confidentialité",
    "privacy_link": "/fr/privacy",
    "privacy_info": "Votre adresse e-mail et votre message seront enregistrés afin que je puisse y répondre. Vos données ne seront pas utilisées à d'autres fins.",
    "submit": "Envoyer",
    "email_error": "Veuillez entrer une adresse e-mail valide",
    "subject_error": "Veuillez entrer un sujet",
    "message_error": "Veuillez entrer un message",
    "captcha_error": "Veuillez résoudre ce captcha",
    "privacy_error": "Veuillez accepter la politique de confidentialité",
    "success": "Merci pour votre message ! Nous vous répondrons dès que possible.",
    "error": "Malheureusement, une erreur inattendue s'est produite. Veuillez réessayer ultérieurement ou envoyez-nous un e-mail à support@letter-app.com.",
    "cookie": "Veuillez accepter les cookies en bas de l'écran pour envoyer un message ou envoyez un e-mail à support@letter-app.com."
  },
  "footer": {
    "rights": "tous droits réservés.",
    "imprint": "Mentions légales",
    "imprint_link": "/fr/imprint",
    "privacy": "Politique de confidentialité",
    "privacy_link": "/fr/privacy"
  },
  "cookie_consent": {
    "message": "Ce site web utilise des cookies pour garantir que vous obtenez la meilleure expérience sur notre site web. Pour en savoir plus, veuillez vous référer à notre",
    "accept": "Accepter"
  },
  "lost": {
    "title": "Vous êtes perdu ! 404 - Page non trouvée",
    "description": "La page que vous recherchez n'existe pas",
    "button": "Retourner à l'accueil"
  },
  "help": {
    "menu": "Aide",
    "link": "aide",
    "title": "Aide",
    "how_to_title": "Comment écrire une lettre avec l'application letter",
    "how_to_001": "Pour commencer à composer votre lettre, appuyez simplement sur le bouton plus situé dans le coin supérieur droit de l'écran principal. Une fois que vous aurez appuyé sur le bouton, un nouvel écran s'ouvrira et vous pourrez y rédiger votre lettre en toute simplicité",
    "how_to_002": "Sur l'écran de l'éditeur de lettres, saisissez les adresses de l'expéditeur et du destinataire, incluez un objet et composez votre message dans le champ \"Message\". Une fois que vous avez terminé votre lettre, appuyez sur \"Enregistrer\" dans le coin supérieur droit",
    "how_to_003": "Votre lettre est automatiquement formatée et affichée. Pour continuer, vous pouvez soit l'imprimer en tapant sur l'icône de l'imprimante, soit la partager par courriel en utilisant le bouton de partage (l'icône carrée avec une flèche)."
  },
  "blog": {
    "menu": "Blog",
    "title": "Blog - Réflexions, histoires et idées"
  },
  "call_to_action": {
    "text": "Téléchargez l'application de lettres maintenant et commencez à écrire votre première lettre — c'est gratuit!"
  }
}