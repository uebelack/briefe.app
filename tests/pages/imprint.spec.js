import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import messages from '../../i18n';
import Imprint from '../../pages/imprint';

const useRouter = jest.spyOn(require('next/router'), 'useRouter');

useRouter.mockImplementation(() => ({
  pathname: '/',
}));

describe('<Imprint/>', () => {
  it('should render', async () => {
    const { container } = render(
      <Imprint />,
    );
    expect(container).toMatchSnapshot();
  });
});
