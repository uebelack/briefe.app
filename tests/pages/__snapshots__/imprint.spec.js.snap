// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Imprint/> should render 1`] = `
<div>
  <div
    class="page"
  >
    <header
      class="fixed top-0 w-full bg-blue text-white"
    >
      <div
        class="md:container px-4 my-4 fade-in flex flex-col md:flex-row"
      >
        <a
          class="flex items-center transition ease-in-out delay-150 flex-grow"
          href="/"
        >
          <img
            alt="Letter logo"
            data-nimg="1"
            decoding="async"
            height="35"
            loading="lazy"
            src="/logo.svg"
            style="color: transparent;"
            width="50"
          />
          <div
            class="text-2xl ml-2 text-white"
          >
            Letter App
          </div>
        </a>
        <ul
          class="flex md:pl-1 pt-1 md:pt-1.5 "
        >
          <li
            class="mr-2"
          >
            <a
              href="/undefined/#details"
            >
              Details
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/undefined/#features"
            >
              Features
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/undefined/#contact"
            >
              Contact
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/undefined/faq"
            >
              FAQ
            </a>
          </li>
          <li>
            |
          </li>
          <li>
            <div
              class="relative"
            >
              <button
                aria-expanded="false"
                aria-haspopup="listbox"
                class="relative w-full cursor-default pl-3 pr-10 text-left w-40"
                data-headlessui-state=""
                id="headlessui-listbox-button-:r0:"
                type="button"
              >
                <span
                  class="pointer-events-none absolute inset-y-0 left-1 flex items-center pr-2"
                >
                  <svg
                    aria-hidden="true"
                    class="h-5 w-5 text-gray-400"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16.555 5.412a8.028 8.028 0 00-3.503-2.81 14.899 14.899 0 011.663 4.472 8.547 8.547 0 001.84-1.662zM13.326 7.825a13.43 13.43 0 00-2.413-5.773 8.087 8.087 0 00-1.826 0 13.43 13.43 0 00-2.413 5.773A8.473 8.473 0 0010 8.5c1.18 0 2.304-.24 3.326-.675zM6.514 9.376A9.98 9.98 0 0010 10c1.226 0 2.4-.22 3.486-.624a13.54 13.54 0 01-.351 3.759A13.54 13.54 0 0110 13.5c-1.079 0-2.128-.127-3.134-.366a13.538 13.538 0 01-.352-3.758zM5.285 7.074a14.9 14.9 0 011.663-4.471 8.028 8.028 0 00-3.503 2.81c.529.638 1.149 1.199 1.84 1.66zM17.334 6.798a7.973 7.973 0 01.614 4.115 13.47 13.47 0 01-3.178 1.72 15.093 15.093 0 00.174-3.939 10.043 10.043 0 002.39-1.896zM2.666 6.798a10.042 10.042 0 002.39 1.896 15.196 15.196 0 00.174 3.94 13.472 13.472 0 01-3.178-1.72 7.973 7.973 0 01.615-4.115zM10 15c.898 0 1.778-.079 2.633-.23a13.473 13.473 0 01-1.72 3.178 8.099 8.099 0 01-1.826 0 13.47 13.47 0 01-1.72-3.178c.855.151 1.735.23 2.633.23zM14.357 14.357a14.912 14.912 0 01-1.305 3.04 8.027 8.027 0 004.345-4.345c-.953.542-1.971.981-3.04 1.305zM6.948 17.397a8.027 8.027 0 01-4.345-4.345c.953.542 1.971.981 3.04 1.305a14.912 14.912 0 001.305 3.04z"
                    />
                  </svg>
                </span>
                <span
                  class="block truncate pl-4"
                >
                  English
                </span>
                <span
                  class="pointer-events-none absolute inset-y-0 right-0 flex items-center pr-2"
                >
                  <svg
                    aria-hidden="true"
                    class="h-5 w-5 text-gray-400"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M10 3a.75.75 0 01.55.24l3.25 3.5a.75.75 0 11-1.1 1.02L10 4.852 7.3 7.76a.75.75 0 01-1.1-1.02l3.25-3.5A.75.75 0 0110 3zm-3.76 9.2a.75.75 0 011.06.04l2.7 2.908 2.7-2.908a.75.75 0 111.1 1.02l-3.25 3.5a.75.75 0 01-1.1 0l-3.25-3.5a.75.75 0 01.04-1.06z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </li>
        </ul>
      </div>
    </header>
    <div
      class="section"
    >
      <div
        class="prose proese-briefe md:container px-4"
      >
        <h1>
          Imprint
        </h1>
        <h2>
          Contact
        </h2>
        <p>
          Übelacker Solutions GmbH
          <br />
          Bläsiring 28
          <br />
          4057 Basel
          <br />
          Switzerland
        </p>
        <p>
          E-Mail: support@briefe.app
          <br />
          Webseite: briefe.app
        </p>
        <h2>
          Authorized representative
        </h2>
        <p>
          David Übelacker
        </p>
        <h2>
          Commercial register entry
        </h2>
        <h3>
          Registered company name:
        </h3>
        <p>
          Übelacker Solutions GmbH
        </p>
        <h3>
          Company No. (UID):
        </h3>
        <p>
          CHE-175.183.669
        </p>
        <h3>
          VAT number
        </h3>
        <p>
          CHE-175.183.669 MWST
        </p>
        <h2>
          Disclaimer
        </h2>
        <p>
          The author assumes no responsibility for the correctness, accuracy, timeliness, reliability and completeness of the information.
        </p>
        <p>
          Liability claims regarding damage caused by the use of any information provided, including any kind of information which is incomplete or incorrect, will therefore be rejected.
        </p>
        <p>
          All offers are non-binding. The author expressly reserves the right to change, supplement or delete parts of the pages or the entire offer without special notice, or to cease publication temporarily or permanently.
        </p>
        <h3>
          Disclaimer for links
        </h3>
        <p>
          References and links to third party websites are outside our area of responsibility. Any responsibility for such websites is disclaimed. The access and use of such websites is at the own risk of the respective user.
        </p>
        <h3>
          Copyright
        </h3>
        <p>
          The copyright and all other rights to the content, images, photos or other files on this website, belong exclusively to the company Übelacker Solutions GmbH or the specifically named rights holders. For the reproduction of any elements, the written consent of the copyright holder must be obtained in advance.
        </p>
      </div>
    </div>
    <footer
      class="md:container mt-20 mb-20 text-center"
    >
      <div
        class="mb-2 px-4 my-4"
      >
        ©
         
        2023
         
        Übelacker Solutions GmbH
        <br />
        <a
          href="/en/imprint"
        >
          Imprint
        </a>
         
        |
         
        <a
          href="/en/privacy"
        >
          Privacy Policy
        </a>
      </div>
    </footer>
  </div>
</div>
`;
